src/tests/test_config.py

```diff
@@ -146,87 +148,106 @@ def test_magic_initialization_with_corrupted_pyproject(tmp_empty, ip_no_magics,
-""",
-            [
-                "Found pyproject.toml from '%s'",
-                "Settings changed:",
-                r"autocommit\s*\|\s*False",
-                r"autolimit\s*\|\s*1",
-                r"style\s*\|\s*RANDOM",
-            ],
-            {"autocommit": False, "autolimit": 1, "style": "RANDOM"},
-        ),
+"""
+    )
+
+    expect = [
+        "Loading configurations from {path}",
+        "Settings changed:",
+        r"autocommit\s*\|\s*False",
+        r"autolimit\s*\|\s*1",
+        r"style\s*\|\s*RANDOM",
+    ]
+
+    config_expected = {"autocommit": False, "autolimit": 1, "style": "RANDOM"}
+
+    toml_path = str(Path(os.getcwd()).joinpath("pyproject.toml"))
+
+    os.mkdir("sub")
+    os.chdir("sub")
+
+    load_ipython_extension(ip_no_magics)
+    magic = ip_no_magics.find_magic("sql").__self__
+    combined = {**get_default_testing_configs(magic), **config_expected}
+    out, _ = capsys.readouterr()
+    expect[0] = expect[0].format(path=re.escape(toml_path))
+    assert all(re.search(substring, out) for substring in expect)
+    assert get_current_configs(magic) == combined
+
+
+@pytest.mark.parametrize(
+    "file_content, param",
+    [
         (
             """
 [tool.jupysql.SqlMagic]
 """,
-            ["Found pyproject.toml from '%s'"],
-            {},
+            "[tool.jupysql.SqlMagic] present in {path} but empty.",
         ),
+        ("", "Tip: You may define configurations in {path}."),
+    ],
+)
+def test_loading_pyproject_toml_display_configuration_docs_link(
+    tmp_empty, ip_no_magics, file_content, param, monkeypatch
+):
+    Path("pyproject.toml").write_text(file_content)
+    toml_path = str(Path(os.getcwd()).joinpath("pyproject.toml"))
+
+    os.mkdir("sub")
+    os.chdir("sub")
+    mock = Mock()
+    monkeypatch.setattr(display, "message_html", mock)
+
+    load_ipython_extension(ip_no_magics)
+    param = (
+        f"{param.format(path=toml_path)} Please review our "
+        f"<a href='{CONFIGURATION_DOCS_STR}'>configuration guideline</a>."
+    )
+    mock.assert_called_once_with(param)
+
+
+@pytest.mark.parametrize(
+    "file_content",
+    [
         (
             """
 [test]
 github = "ploomber/jupysql"
-""",
-            ["Found pyproject.toml from '%s'"],
-            {},
+"""
         ),
         (
             """
 [tool.pkgmt]
 github = "ploomber/jupysql"
-""",
-            ["Found pyproject.toml from '%s'"],
-            {},
+"""
         ),
         (
             """
 [tool.jupysql.test]
 github = "ploomber/jupysql"
-""",
-            ["Found pyproject.toml from '%s'"],
-            {},
-        ),
-        (
-            "",
-            ["Found pyproject.toml from '%s'"],
-            {},
+"""
         ),
     ],
 )
-def test_loading_valid_pyproject_toml_shows_feedback_and_modifies_config(
-    tmp_empty,
-    ip_no_magics,
-    capsys,
-    file_content,
-    expect,
-    config_expected,
+def test_load_pyproject_toml_configuration_message_not_displayed(
```

Comment:
this test is unclear, why are we expecting the message not to be displayed? please modify the test name to reflect that